selfdrive/updated.py

```diff
@@ -73,15 +75,14 @@ def graceful_shutdown(self, signum, frame):
 
   def update_now(self, signum, frame):
     cloudlog.info("caught SIGHUP, running update check immediately")
+    #global DEBUG_FORCE_UPDATE
+    #DEBUG_FORCE_UPDATE = True
     self.ready_event.set()
 
 
 def wait_between_updates(ready_event):
   ready_event.clear()
-  if SHORT:
-    ready_event.wait(timeout=10)
-  else:
-    ready_event.wait(timeout=60 * 10)
+  ready_event.wait(timeout=WAIT_BETWEEN_ATTEMPTS)
```

Comment:
this function is only called in one place, don't think it makes sense to make this a function